nucliadb_sdk/nucliadb_sdk/v2/sdk.py

```diff
@@ -846,18 +846,22 @@ async def _request(
         method: str,
         data: Optional[Union[str, bytes]] = None,
         query_params: Optional[Dict[str, str]] = None,
+        content: Optional[RawRequestContent] = None,
     ):
         url = f"{self.base_url}{path}"
         opts: Dict[str, Any] = {}
+        if all([data, content]):
+            raise ValueError("Cannot provide both data and content")
         if data is not None:
             opts["data"] = data
+        if content is not None:
+            opts["content"] = content
         if query_params is not None:
             opts["params"] = query_params
         response: httpx.Response = await getattr(self.session, method.lower())(
             url, **opts
         )
-        self._check_response(response)
-        return response
+        return self._check_response(response)
```

Comment:
Careful here! 😜 the response should be returned

Reply:
`self._check_response(response)` is also returning the response, I followed a similar syntax as the sync method 😁: https://github.com/nuclia/nucliadb/blob/main/nucliadb_sdk/nucliadb_sdk/v2/sdk.py#L747-L766

Reply:
oh you're right! 👍🏼